mobile/lib/features/wallet/receive_screen.dart

```diff
@@ -470,7 +470,8 @@ class SelectableButton extends StatelessWidget {
       onPressed: onPressed,
       style: ButtonStyle(
         iconSize: MaterialStateProperty.all<double>(20.0),
-        elevation: MaterialStateProperty.all<double>(0), // this reduces the shade
+        elevation: MaterialStateProperty.all<double>(0),
+        // this reduces the shade
```

Comment:
🔧 I think the comment belongs next to or on top of the previous line. Same thing happened below.